x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartTrainedModelDeploymentAction.java

```diff
@@ -302,6 +329,10 @@ public ActionRequestValidationException validate() {
             if (threadsPerAllocation < 1) {
                 validationException.addValidationError("[" + THREADS_PER_ALLOCATION + "] must be a positive integer");
             }
+            ActionRequestValidationException autoscaleException = autoscalingSettings == null ? null : autoscalingSettings.validate();
```

Comment:
No, that's fine. The behaviour is:
- set the initial number of allocations to `numAllocations` (or 1, which is the default)
- if it's out of bounds for the autoscaling, it will fix it in the first cycle

I think that's simple and explainable behaviour (and doesn't require additional code and verification)